src/icons/icon.ts

```diff
@@ -21,7 +21,10 @@ export class NglIcon {
     if (this.size) {
       classes.push(`slds-icon--${this.size}`);
     }
-    classes.push(`slds-icon-text-${this.type || 'default'}`);
+
+    if (this.type) {
+      classes.push(`slds-icon-text-${this.type}`);
+    }
 
```

Comment:
If I am reading this correctly, if ```type``` is instantiated with ```'default'``` and we omit the ```[type]``` binding, then wouldn't we still apply the ```slds-icon-text-default``` class, by default?